### PR TITLE
feat(ui-tests): cover ethers

### PR DESCRIPTION
**Pull request by arein**
# Breaking Changes

N/A

# Changes

- feat(ui-tests): cover ethers

# Associated Issues

closes #...

### CODE REVIEW

**Review by chris13524 — DISMISSED** [review]
Very cool approach!

File: apps/laboratory/tests/shared/constants/index.ts [hunk 1]
```diff
@@ -1,7 +1,9 @@
 import type { SessionParams } from '../types'
 
 // Allow localhost
-export const LOCAL_LABS_URL = 'http://localhost:3000/library/wagmi/'
+export const BASE_URL = 'http://localhost:3000/library/'
+// Allow localhost
+export const LOCAL_LABS_URL = `${BASE_URL}wagmi/`
```
Comment (tomiir) [inline apps/laboratory/tests/shared/constants/index.ts:6]: why set the base url to wagmi?

Comment (arein) [reply apps/laboratory/tests/shared/constants/index.ts:6]: `BASE_URL` now is not specific. `LOCAL_LABS_URL` is the entrypoint for the test and needs to be a website that doesn't HTTP404 so I chose this one. Could also use `http://localhost:3000/` I presume

Comment (arein) [reply apps/laboratory/tests/shared/constants/index.ts]: Fixed

**Review by tomiir — APPROVED** [review]

**Pull request merged**